src/test/java/org/jabref/logic/importer/fileformat/pdf/PdfMergeMetadataImporterTest.java

```diff
@@ -85,7 +87,34 @@ void importWorksAsExpected() throws Exception {
         // From merge
         expected.setFiles(List.of(new LinkedFile("", file.toAbsolutePath(), StandardFileType.PDF.getName())));
 
-        assertEquals(Collections.singletonList(expected), result);
+        assertEquals(List.of(expected), result);
+    }
+
+    @Test
+    void pdfMetadataExtractedFrom2024SPLCBecker() throws Exception {
+        Path file = Path.of(PdfMergeMetadataImporterTest.class.getResource("2024_SPLC_Becker.pdf").toURI());
+        List<BibEntry> result = importer.importDatabase(file).getDatabase().getEntries();
+
+        BibEntry expected = new BibEntry(StandardEntryType.InProceedings)
+                .withCitationKey("Becker_2024")
+                .withField(StandardField.AUTHOR, "Becker, Martin and Rabiser, Rick and Botterweck, Goetz")
+                .withField(StandardField.TITLE, "Not Quite There Yet: Remaining Challenges in Systems and Software Product Line Engineering as Perceived by Industry Practitioners")
+                .withField(StandardField.BOOKTITLE, "28th ACM International Systems and Software Product Line Conference")
+                .withField(StandardField.SERIES, "SPLC ’24")
+                .withField(StandardField.PUBLISHER, "ACM")
+                .withField(StandardField.YEAR, "2024")
+                .withField(StandardField.MONTH, "#sep#")
+                .withField(StandardField.PAGES, "179--190")
+                .withField(StandardField.DOI, "10.1145/3646548.3672587")
+                .withField(StandardField.KEYWORDS, "case studies about companies adopting product line engineering")
+                .withField(FieldFactory.parseField("collection"), "SPLC ’24")
+                .withField(StandardField.FILE, ":" + file.toString().replace("\\", "/").replace(":", "\\:") + ":PDF")
+
+                // TODO: Abstract not correct yet --> parsing logic needs to be improved
+                // Abstract is CC-BY, thus no issue to include it here.
```

Comment:
ToDo?

Reply:
It was wrong before. Left for future work.